rust/common-bundler/src/lib.rs

```diff
@@ -147,6 +147,7 @@ pub mod tests {
 
     #[tokio::test]
     async fn it_bundles_javascript_from_url() -> Result<()> {
+        common_tracing::initialize_tracing();
```

Comment:
Would you be willing to take a stab at a procedural macro that we can use to decorate tests?